### ISSUE TITLE
context-in-play-function support only "context" as a variable name

### ISSUE THREAD

**Issue by beaussan** [storybookjs/eslint-plugin-storybook#118]
**Describe the bug**
Context in play function only support only "context" as a variable names. Any other variable names gives a lint error. 

**To Reproduce**
Given this code : 
```ts
const Story1 = Template.bind({});
Story1.play = async (ctx) => {
...
}

const Story2 = Template.bind({});
Story2.play = async (ctx) => {
	await Story1.play(ctx)
}
```
the lint should not fail since we are passing the context, but with a different variable name

**Expected behavior**
There should not have any lint errors since it respects the lint rule.

**Comment by yannbf** [comment]
Thanks for opening this issue @beaussan! That makes total sense. Are you willing to make a PR for this?
This is the place where you'd need to touch: https://github.com/storybookjs/eslint-plugin-storybook/blob/main/lib/rules/context-in-play-function.ts#L70

**Comment by beaussan** [comment]
I'll try yes ! I saw the file when I opened the issue and I should have some time today to at least give it a try.
I know a bit of ast / eslint knoledge, but the thing I don't know how to achieve is how to get the the name of the argument of the current function.

My first idea is to try to go up the ast to get the function itself, but that maybe flaky

**Comment by yannbf** [comment]
I always recommend trying https://astexplorer.net/ out, it helps you immensely when figuring out AST structures! And from there you can start playing around 🎡

**Comment by Dschungelabenteuer** [comment]
@beaussan LMK if you need any help from a french fellow

**Comment by beaussan** [comment]
Thanks to the both of you !

I finally got some time to work on this, and made a PR :tada:

**Comment by yannbf** [comment]
A fix for this was released in [v0.6.11](https://github.com/storybookjs/eslint-plugin-storybook/releases/tag/v0.6.11) 🚀

Thanks @beaussan !!
